E2_ESRGAN/lib/model.py

```diff
@@ -46,3 +45,29 @@ def call(self, input_):
     feature = self.lrelu(self.conv_last_1(feature))
     out = self.conv_last_2(feature)
     return out
+
+
+class VGGArch(tf.keras.Model):
```

Comment:
Lets add docstrings, here and to the previous class.

Reply:
I've addressed adding docstrings as one of the tasks, #23, Will be adding them when I address this PR. What do you suggest?

Reply:
SG.